.travis.yml

```diff
@@ -73,6 +73,10 @@ matrix:
       env:
         - JOB="3.6_NUMPY_DEV" TEST_ARGS="--skip-slow --skip-network" PANDAS_TESTING_MODE="deprecate"
     # In allow_failures
+    - dist: trusty
```

Comment:
move this after the doc build

Reply:
@jorisvandenbossche prefered this before the doc build

https://github.com/pandas-dev/pandas/pull/19236#discussion_r162569822